### DISCUSSION TITLE
Path doesn't Exist

### DISCUSSION THREAD

**Discussion by 0xmillysmith** [Cyfrin/foundry-full-course-cu#1380] [Q&A] [answered]
<img width="948" alt="image" src="https://github.com/Cyfrin/foundry-full-course-f23/assets/99047908/4b6b5df8-d9a4-4363-ad31-6a124643fa72">

Correct Path thanks

**Comment by 0xusmanf** [comment]
https://github.com/smartcontractkit/chainlink/tree/v2.7.2/contracts/src/v0.8/interfaces

**Comment by 0xmillysmith** [comment, marked as answer]
Thanks Usman

**Comment by 0xmillysmith** [comment]
Issue Resolved.
